File: src/main/java/com/procurement/access/model/dto/lots/LotsRequestDto.java
package com.procurement.access.model.dto.lots;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;
import lombok.Getter;

@Getter
public class LotsRequestDto {

    @JsonProperty(value = "unsuccessfulLots")
    private List<LotDto> lots;

    public LotsRequestDto(@JsonProperty("unsuccessfulLots") final List<LotDto> lots) {
        this.lots = lots;
    }

}

File: src/main/kotlin/com/procurement/access/model/dto/lots/LotsRequestDto.kt
package com.procurement.access.model.dto.lots

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class LotsRequestDto @JsonCreator constructor(

        val unsuccessfulLots: HashSet<LotDto>?
)
